config/serverless.oblt.yml

```diff
@@ -25,6 +25,8 @@ xpack.features.overrides:
   ### Discover feature should be moved from Analytics category to the Observability one and its privileges are
   ### fine-tuned to grant access to Observability app.
   discover_v2.category: 'observability'
+  ### Agent Builder should be moved from Analytics category to the Observability one.
+  agentBuilder.category: 'observability'
```

Comment:
Do we intentionally keep `Agent Builder` under `Analytics` in the Chat project type and as the only item in that category for Search project type?

<img width="904" height="1232" alt="image" src="https://github.com/user-attachments/assets/47f249cd-e555-40eb-b0b8-7f281e32c41e" />

Also, not related to this PR, but `Workchat Framework` doesn't look right

<img width="904" height="198" alt="Screenshot 2025-09-10 at 09 02 17" src="https://github.com/user-attachments/assets/d0a42e07-b916-43fc-a0e9-0b6160b92847" />

Reply:
For the chat project type yes (for now).
For search definitely not, thank you I'll fix that.